Dockerfile

```diff
@@ -22,12 +8,8 @@ ARG MB_EDITION=oss
 
 WORKDIR /home/circleci
 
-# try to reuse caching as much as possible
-COPY --from=frontend /home/circleci/.m2/repository/. /home/circleci/.m2/repository/.
-COPY --from=frontend /home/circleci/. .
-
-# build the app
-RUN INTERACTIVE=false MB_EDITION=$MB_EDITION bin/build version drivers uberjar
+COPY --chown=circleci . .
+RUN INTERACTIVE=false CI=true MB_EDITION=$MB_EDITION bin/build
```

Comment:
@paoliniluis the only way I can make it work is by merging these builder stages into two. I think this will be sub-optimal for Buildkit parallelism perspective, but maybe it does not matter much anymore now that the overall build is much faster anyway (after moving away from Leiningen).

Reply:
100% agree, and also the simpler the better